apollo/apollo.go

```diff
@@ -206,38 +197,45 @@ func (c *client) RegisterConfigCallback(param ConfigParam,
 		callback(data, c.parser)
 	}
 
-	configMap := c.acli.GetNameSpace(param.NameSpace)
+	configMap := c.acli.GetNameSpace(param.nameSpace)
 	data, ok := configMap[param.Key]
 	if !ok {
 		klog.Info("key:", param.Key)
 		klog.Info("CONFIG:", configMap)
-		panic(errors.New("kitex config : key not found"))
+		klog.Debug("[apollo] key not found")
+	} else {
+		callback(data.(string), c.parser)
 	}
-	// data := c.acli.Get(param.Key, agollo.WithNamespace(param.NameSpace))
-	callback(data.(string), c.parser)
 
 	go c.listenConfig(param, onChange)
 }
 
 func (c *client) listenConfig(param ConfigParam, callback func(namespace, cluster, key, data string)) {
+	defer func() {
+		if err := recover(); err != nil {
+			klog.Error("[apollo] listen goroutine error:", err)
+		}
+	}()
 	errorsCh := c.acli.Start()
-	apolloRespCh := c.acli.WatchNamespace(param.NameSpace, make(chan bool))
+	apolloRespCh := c.acli.WatchNamespace(param.nameSpace, make(chan bool))
 
 	for {
 		select {
 		case resp := <-apolloRespCh:
+			klog.Info("[apollo] config update")
 			data, ok := resp.NewValue[param.Key]
 			if !ok {
+				// Deal with delete config
```

Comment:
The error log is not appropriate，the lack of key may meet the user's expectation,  we should not force the user to set every policy.